src/Commands/BackupCommand.php

```diff
@@ -47,6 +51,12 @@ public function handle()
                 $backupJob->setFilename($this->option('filename'));
             }
 
+            if ($this->option('tries')) {
+                $this->tries = (int)$this->option('tries');
+            } elseif (!empty(config('backup.backup.tries'))) {
```

Comment:
I really dislike `else` statements. Could you refactor this to a dedicated method with early returns?